### DISCUSSION TITLE
RAVEN for Inverse Uncertainty Quantification (IUQ)

### DISCUSSION THREAD

**Discussion by JohninSJ** [idaholab/raven#2247] [Q&A]
I am wondering if RAVEN is capable for doing Inverse Uncertainty Quantification (IUQ). If yes, what methodologies are used, and where the application examples can be obtained. Could someone provide associated information? Much appreciated.

**Comment by wangcj05** [comment]
@JohninSJ FYI, we have Markov chain Monte Carlo algorithm available for IUQ analysis. You can check tests under "raven/tests/framework/MCMC" for more information. In addition, we have made a raven plugin that is mainly for IUQ, and you can check at "https://github.com/idaholab/BayCal", and you can find examples for fission gas release model calibrations (i.e., IUQ) at: "https://github.com/idaholab/BayCal/tree/main/examples/fgr".

**Reply by JohninSJ** [reply]
@wangcj05 Thanks you very much for your helpful information. Will look at the links to obtain details of RAVEN IUQ methodologies and applications. I am working in CNL and going to do some investigations on applying IUQ to code validations against experimental data. The first step is to see what existing IUQ tools applicable for the investigation purposes.
